acceptance/tests/rototillerTask/command_arguments_with_override.rb

```diff
@@ -27,6 +29,7 @@
     task_name    = test_filename + "#{tasks.length}"
     override_env = test_filename.upcase + random_string
 
+    #this task has no default for the ENV I think this should stop
```

Comment:
why wouldn't it take the name of the parent (name of the argument) as the env default?